public/assets/scss/_styles.scss

```diff
@@ -352,7 +407,198 @@ h5 .btn-link {
         font-size: 0.8rem;
     }
 }
+.logo-div {
+    background-color: $gray-300;
+    padding: 3rem 1rem 2rem 5rem;
+
+    &-text {
+        padding-top: 2rem;
+        color: $gray-700;
+        font-weight: 300;
+        font-size: 0.8rem;
+    }
+}
 .sm-icon {
     width: 1.8rem;
     height: 1.8rem;
 }
+
+
+/* big tablets */
+@media only screen and (max-width: 1024px) {
```

Comment:
I think it would be better to add the responsive styles inside the respective classes instead of adding them separately. This will make it easier to update as opposed to looking for the responsive styles in another section. Here is an example of doing it the right way:

https://github.com/Highrachy/ballers-ws/blob/27109479af05ba60037f4390f1c3219770d287d8/public/assets/scss/_ballers.scss#L33-L51

* On line 38, a media query is added for media screen and above.
* On line 44, another media query is added for the large screen above.
This will make it easier to update all the changes to the H1 tag without searching for the screen size.